.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
- [ ] This project is using JavaScript, please consider using the [right linters config files](https://github.com/microverseinc/linters-config/tree/master/html-css-js) so the linters checked for Eslint (JavaScript)